src/palsetup/main.cpp

```diff
@@ -28,36 +29,52 @@ void writeConfigFile(const std::string& outputDir,
                      const OHLCTimeSeries<Num>& outOfSampleSeries,
                      const std::string& timeFrame)
 {
-    std::string configFileName = outputDir + "/" + tickerSymbol + "_config.csv";
-    std::ofstream configFile(configFileName);
-    if (!configFile.is_open()) {
-        std::cerr << "Error: Unable to open config file " << configFileName << std::endl;
-        return;
+  std::string configFileName = outputDir + "/" + tickerSymbol + "_config.csv";
```

Comment:
nit: Might be worth to use `fs::path` instead of a `std::string` here

Reply:
Requested change made in latest commit